cloud_testing/xctest_parser/src/test/kotlin/xctest/ShardTest.kt

```diff
@@ -0,0 +1,11 @@
+package xctest
+
+import org.junit.Test
+
+class ShardTest {
+
+    @Test
+    fun shardWorks() {
+        // TODO: test this without committing a giant binary blob to the git repo
```

Comment:
good point. Can't you build the iOS app before the tests run?

Reply:
The issue is the iOS app isn't even in this repo. I think we'll need to build a custom download logic to get all the pieces in place for unit testing. I'm hoping I get approval from Walmart to buy Git LFS since that's a better solution to binaries in git repos.

Reply:
mmm what about using git submodules for the iOS repo?

Reply:
issue with submodules is that the binary files still end up in a repo that gets bigger each time. I'll probably upload a zip as a GitHub release to a new repo and download that on demand.

Reply:
I uploaded the test zip here:
https://github.com/Flank/test_artifacts/releases/tag/test